packages/cli/src/commands/start.ts

```diff
@@ -198,6 +198,42 @@ async function askQuestions({ template: defaultTemplate }: Pick<StartOptions, "t
       process.exit(0);
     }
     context.services = services;
+
+    // Ask for environment variables
+    const envList = Array.from(new Set(connectors.filter(c => services.includes(c.serviceName)).flatMap((c) => {
+      return c.envList;
+    })));
+
+    const isConfirm = await p.confirm({
+      message: `Do you want to enter environment variables? (Number of environment variables to enter: about ${envList.length})`,
+      initialValue: false,
+    });
+
+    const envInfos: EnvInfo[] = [];
+
+    if (p.isCancel(isConfirm) || !isConfirm) {
+      p.cancel("Skipping environment variables input.");
+    }
+    else {
+      for (const env of envList) {
+        const envValue = await p.text({
+          message: `${env}: `,
+          defaultValue: "",
+        });
+
+        if (p.isCancel(envValue)) {
+          p.cancel("Skipping environment variables input.");
+          break;
+        }
+
+        envInfos.push({
+          name: env,
+          value: typeof envValue === "string" ? envValue : "",
```

Comment:
this typeguard is meaningless because `p.isCancel` already process typeguard
https://github.com/bombshell-dev/clack/blob/main/packages/core/src/utils/index.ts#L16-L18

Reply:
you're genius